src/wallet/wallet.h

```diff
@@ -571,6 +575,9 @@ class CWallet : public CCryptoKeyStore, public CValidationInterface
 
     const CWalletTx* GetWalletTx(const uint256& hash) const;
 
+    // MVHF-BU
```

Comment:
this new method should trace to what is now section 5.5.3 (Procedure to create the wallet file backup) in the design.md.

We need to add design elements to 5.5.2 and 5.5.3 , and then trace them here.

Don't worry about this now, I'm just noting it for later.